Spanner/src/Database.php

```diff
@@ -920,7 +920,11 @@ public function runTransaction(callable $operation, array $options = [])
      *
      * @param string $table The table to mutate.
      * @param array $data The row data to insert.
-     * @param array $options [optional] Configuration options.
+     * @param array $options [optional] {
+     *     Configuration options.
+     *
+     *     @type array $requestOptions Request options.
```

Comment:
What are your thoughts on adding the following to the description?

```
For more information on available options, please see
[the upstream documentation](https://cloud.google.com/spanner/docs/reference/rest/v1/RequestOptions).
Please note, if using the `priority` setting you may utilize the constants available
on {@see Google\Cloud\Spanner\V1\RequestOptions\Priority} to set a value.
```

Reply:
I think it's an excellent addition! I've updated the PR to include this change :+1: